config/generic-mellow-fly-e3-v2.cfg

```diff
@@ -0,0 +1,234 @@
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
```

Comment:
Typically example config start with a description, but the first two lines of this file have an unusual copyright statement.

-Kevin

Reply:
Oh you are totally right, in my head that was normal and in all configs for some reason! False memory

Reply:
Hi,

This is most likely the result of some copypasta. If you want I can remove the offending lines